packages/data-provider/src/utils/types.ts

```diff
@@ -30,6 +30,60 @@ export const AUDIT_ACTION = {
 } as const;
 export type AuditAction = ObjectValues<typeof AUDIT_ACTION>;
 
+/**
+ * Audit Raw Data from Repository
+ */
+export type AuditDataRepository = {
```

Comment:
Being picky... This isnt the type for the Repository, its the type for a Repository record. In my java days they would call this the DAO (data access object). Would you consider the name `AuditRepositoryRecord`

Reply:
renamed to `AuditRepositoryRecord `